bundle/tests/python_wheel/python_wheel_no_artifact_notebook/bundle.yml

```diff
@@ -0,0 +1,15 @@
+bundle:
+  name: python-wheel-notebook
```

Comment:
Directory names are flipped between this and the `python-wheel` bundle.

Reply:
Good catch! :)